public/index.html

```diff
@@ -186,6 +186,146 @@ <h6 class="card-title text-center py-1" style="background-color: black;">Creator
+                </div>
+            </div>
+            <div class="form-group row mt-2">
+                <label for="input" class="col-sm-2 col-form-label">Title</label>
+                <div class="alert alert-info form-inline col-sm-10" role="alert">
+                    <div id="#"></div>
+                </div>
+            </div>
+            <div class="form-group row">
+                <label for="description" class="col-sm-2 col-form-label">Description</label>
+                <div class="alert alert-info form-inline col-sm-10" role="alert">
+                    <div id="#"></div>
+                </div>
+            </div>
+
+            <div class="form-row">
+                <div class="form-group col-md-4">
+                    <label for="input">Assignee</label>
+                    <div class="alert alert-info form-inline" role="alert">
+                        <div id="#"></div>
+                    </div>
+                </div>
+                <div class="form-group col-md-4">
+                    <label for="text">Creator</label>
+                    <div class="alert alert-info form-inline" role="alert">
+                        <div id="#"></div>
+                    </div>
+                </div>
+                <div class="form-group col-md-4">
+                    <label for="text">Estimated Time</label>
+                    <div class="alert alert-info form-inline" role="alert">
+                        <div id="#"></div>
+                    </div>
+                </div>
+            </div>
+            <div class="form-row">
+                <div class="form-group col-md-4">
+                    <label for="input">Category</label>
+                    <div class="alert alert-info form-inline" role="alert">
+                        <div id="#"></div>
+                    </div>
+                </div>
+                <div class="form-group col-md-4">
+                    <label for="text">Priority</label>
+                    <div class="alert alert-info form-inline" role="alert">
+                        <div id="#"></div>
+                    </div>
+                </div>
+                <div class="form-group col-md-4">
+                    <label for="text">Difficulty</label>
+                    <div class="alert alert-info form-inline" role="alert">
+                        <div id="#"></div>
+                    </div>
+                </div>
+            </div>
+            <div class="form-row">
+                <div class="form-group col-md-4">
+                    <label for="inputState">Status</label>
+                    <div class="alert alert-info form-inline" role="alert">
+                        <div id="#"></div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <!-- log work-------------------------------------------------------- -->
+
+    <div class="container bg-dark py-2 px-5 mb-5">
```

Comment:
Please Add comment lines and mention the py px and mb there, So that others can understand what they do.

Reply:
this is the basic bootstrap syntax...I will add the comment line.